### PR TITLE
Added my username to REVIEWERS

### PR DESCRIPTION
**Pull request by ayanujju**
Added my username to REVIEWERS

### CODE REVIEW

File: site/fr/REVIEWERS [hunk 1]
```diff
@@ -10,3 +10,4 @@ TahiriNadia
 bayethiernodiop
 SamuelAlev
 gatarelib
+Ayan Raj
```
Comment (SamuelAlev) [inline site/fr/REVIEWERS]:
```suggestion
ayanujju
```

Comment (ayanujju) [reply site/fr/REVIEWERS]: Done

**Comment by ayanujju** [comment]
French is not my spoken language but I have learned its basics in my course curriculum.

**Comment by TahiriNadia** [comment]
> French is not my spoken language but I have learned its basics in my course curriculum.

If you are an intermediate level (minimum) in French on writing it'll be OK.

**Pull request merged**